cltk/corpus/utils/importer.py

```diff
@@ -1,392 +1,394 @@
-                    logger.info(msg)
-                    git_origin.pull()
-                except CorpusImportError as corpus_imp_err:
-                    msg = "Git pull of '{}' failed: '{}'".format(git_uri, corpus_imp_err)
-                    logger.error(msg)
-        elif location == 'local':
-            msg = "Importing from local path: '{}'".format(local_path)
-            logger.info(msg)
-            if corpus_name in ('phi5', 'phi7', 'tlg'):
-                if corpus_name == 'phi5':
-                    # normalize path for checking dir
-                    if local_path.endswith('/'):
-                        local_path = local_path[:-1]
-                    # check for right corpus dir
-                    if os.path.split(local_path)[1] != 'PHI5':
-                        logger.info("Directory must be named 'PHI5'.")
-                if corpus_name == 'phi7':
-                    # normalize local_path for checking dir
-                    if local_path.endswith('/'):
-                        local_path = local_path[:-1]
-                    # check for right corpus dir
-                    if os.path.split(local_path)[1] != 'PHI7':
-                        logger.info("Directory must be named 'PHI7'.")
-                if corpus_name == 'tlg':
-                    # normalize path for checking dir
-                    if local_path.endswith('/'):
-                        local_path = local_path[:-1]
-                    # check for right corpus dir
-                    if os.path.split(local_path)[1] != 'TLG_E':
-                        logger.info("Directory must be named 'TLG_E'.")
-                # move the dir-checking commands into a function
-                data_dir = os.path.expanduser(CLTK_DATA_DIR)
-                originals_dir = os.path.join(data_dir, 'originals')
-                # check for `originals` dir; if not present mkdir
-                if not os.path.isdir(originals_dir):
-                    os.makedirs(originals_dir)
-                    msg = "Wrote directory at '{}'.".format(originals_dir)
-                    logger.info(msg)
-                tlg_originals_dir = os.path.join(data_dir,
-                                                 'originals',
-                                                 corpus_name)
-                # check for `originals/<corpus_name>`; if pres, delete
-                if os.path.isdir(tlg_originals_dir):
-                    shutil.rmtree(tlg_originals_dir)
-                    msg = "Removed directory at '{}'.".format(tlg_originals_dir)
-                    logger.info(msg)
-                # copy_dir requires that target
-                if not os.path.isdir(tlg_originals_dir):
-                    self._copy_dir_recursive(local_path, tlg_originals_dir)
-
-
-if __name__ == '__main__':
-    c = CorpusImporter('latin')
-    # print(c.list_corpora)
-    c.import_corpus('latin_training_set_sentence_cltk')
+"""Import CLTK corpora.

```

Comment:
revert this file as you found it and just add your changes.

Reply:
I just add my changes...but I don't understand why it changes all the lines

Reply:
This is messed up, So, I closed this and but again I added a new PR and it Passed all checks #750